.github/workflows/ci-post-commit-analyzer.yml

```diff
@@ -45,19 +45,12 @@ jobs:
         env:
           DEBIAN_FRONTEND: noninteractive
         run: |
-          apt-get update
-          apt-get -y install \
-            wget \
-            gnupg
-          echo "deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-18 main" | tee -a /etc/apt/sources.list.d/llvm.list
-          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
           apt-get update
           apt-get -y install \
             cmake \
             ninja-build \
             perl \
-            clang-tools-$LLVM_VERSION \
-            clang-$LLVM_VERSION
+            clang-tools
```

Comment:
What's the specific binary that we need from this package? We should probably add a TODO to include it in `LLVM_DISTRIBUTION_COMPONENTS` for the CI container so we can have a hermetic toolchain.

Reply:
We just need `scan-build` which is actually a perl script.

Reply:
@boomanaiden154 #94543

Reply:
You can use `scan-build` directly from the source directory if you like, there's no need to install it with `apt-get`.

Reply:
@haoNoQ OK, I still think I'm going to put it on the container, because that gives us a more stable version to use.

Reply:
I'm running from the source for now until the container is updated.